doc/api/events.md

```diff
@@ -830,7 +830,7 @@ events. Listeners installed using this symbol are called before the regular
 `'error'` listeners are called.
 
 Installing a listener using this symbol does not change the behavior once an
-`'error'` event is emitted, therefore the process will still crash if no
+`'error'` event is emitted. Therefore the process will still crash if no
```

Comment:
```suggestion
`'error'` event is emitted. Therefore, the process will still crash if no
```
Isn't it more common to use a comma after therefore at the beginning of a sentence?